sdk/hdinsight/Azure.ResourceManager.HDInsight.Containers/src/Azure.ResourceManager.HDInsight.Containers.csproj

```diff
@@ -1,6 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Version>1.0.0-beta.3</Version>
```

Comment:
beta.3 is an unreleased version already, we do not need to bump it. Please revert this change.